starknet/src/factory/factory.cairo

```diff
@@ -40,8 +40,14 @@ mod Factory {
             contract_address_salt: felt252,
```

Comment:
we should probably call this a salt nonce now. 

also might as well make it more equivalent to sx-evm by placing the nonce at the end of the args:  https://github.com/snapshot-labs/sx-evm/blob/df37a70673067300a3a6c0541740117a86d0bc59/src/ProxyFactory.sol#L12C86-L12C86

Reply:
yeah original naming was based on the [deploy_syscall argument name](https://github.com/starkware-libs/cairo/blob/775b4f84e705293ded7b7cc203650eb983246842/corelib/src/starknet/syscalls.cairo#L24).. `salt_nonce` sounds weird to me because `nonce` refers to something we increment usually no?

how about just `salt`? :p

Reply:
yeah thats probably what we would do right? if a user wanted to deploy an identical space they would just need to increment the nonce

Reply:
yeah I guess so. In my mind we would just query a random number, because incrementing a nonce requires knowing which nonce was previously used, which sounds more cumbersome than just genereating a random number

ab1004cee63ddf6ffb008a1a36a7109e451f1e62